### ISSUE TITLE
Taskbar becomes unresponsive when right clicked

### ISSUE THREAD

**Issue by Diego-JRH** [valinet/ExplorerPatcher#4073]
### Before reporting your issue

- [X] I have confirmed that this issue does not happen when ExplorerPatcher is not installed
- [X] I do not have "register as shell extension" enabled
- [X] I have tried my best to check existing issues

### Repro ExplorerPatcher versions

Explorer Patcher 22621.4317.67.1

### Repro Windows Versions

Edición	Windows 11 Pro
Versión	24H2
Instalado el	‎09/‎12/‎2024
Versión del sistema operativo	26100.2605
Experiencia	Paquete de experiencia de características de Windows 1000.26100.36.0
x64

### 3rd party tweak software installed

None

### Describe the bug

1. Install Explorer Patcher
2. Set Windows 10 start menu (dont know if this is relevant)
3. Hover the cursor over the taskbar icon, they work as usual, even when clicked
4. Right click on the taskbar
5. Click on the options that appear or click outside to close the menu
6. The taskbar appears as if it has moved to the background.
7. Hover over the taskbar icons and click them, they act as if they are kept pressed
8. Click again in one of the icons of the taskbar or the taskbar itself.
9. The problem solves.

### Expected outcome

When pressing the icons, they should open the program (or open the start menu if the windows icon is pressed) even if you first right click in the taskbar.

### Actual outcome

If a right click in the taskbar is performed, any action in the taskbar fails unless the taskbar is clicked again.

### Additional info

Check video to see the bug

### Media

https://github.com/user-attachments/assets/ceb874b0-8736-431f-aa06-2985e7b2aff0

**Comment by pyrates999** [comment]
To verify this is caused by enabling the windows 10 start menu, change the start menu back to windows 11 start menu.  And check if this still happens.

**Comment by Diego-JRH** [comment]
I just did, and the problem persists, in fact I restored defaults and the problem is still there.

**Comment by pyrates999** [comment]
Let's see what the console in explorer patcher shows.  Open up the properties application for explorer patcher, go to advanced, and click on enable console.

Recreate the issue and copy and paste the output from the console.

**Comment by pyrates999** [comment]
Ok, thanks.  Please wait for a developer to respond.

**Comment by Amrsatrio** [comment]
@Diego-JRH Check C:\Program FIles\ExplorerPatcher. Do you have a file named "ep_taskbar.5.dll"? If you don't, please reinstall EP.

**Comment by Diego-JRH** [comment]
<img width="792" alt="Image" src="https://github.com/user-attachments/assets/2ed3aad1-5b1b-414d-ba93-2f7be2ab1631" />

Yes I have it.

**Comment by galaxy3946** [comment]
I have same problem, too. 
No menu by right click at Windows start menu but left click action is normal.

OS: Windows 11 24H2 Japanese version

non-English language OS specific issue?

**Comment by Amrsatrio** [comment]
Repro on 26100.2605 ARM64.  I need to update Valinet's code that handles the classic context menu. For now you can disable the context menu modification, or use the remade Windows 10 taskbar.

![Image](https://github.com/user-attachments/assets/19a8bfdd-5d1f-4a1e-997e-3ffdd07f0991)

**Comment by pyrates999** [comment]
I run windows 10 19045.5247, and checked the directory `C:\Program FIles\ExplorerPatcher`.  There is no file named `ep_taskbar.5.dll` there either.

**Comment by Diego-JRH** [comment]
> Repro on 26100.2605 ARM64. I need to update Valinet's code that handles the classic context menu. For now you can disable the context menu modification, or use the remade Windows 10 taskbar.
> 
> ![Image](https://github.com/user-attachments/assets/19a8bfdd-5d1f-4a1e-997e-3ffdd07f0991)

Yep, the issue seems to be there, when I enabled that option the issue doesn't happen.
